r/geoarrow/R/wk-compat.R

```diff
@@ -112,6 +112,60 @@ infer_nanoarrow_schema.wk_xy <- function(x, ...) {
   wk_geoarrow_schema(x, na_extension_geoarrow, "POINT", dimensions = dims)
 }
 
+#' @export
+convert_array.wk_wkt <- function(array, to, ...) {
+  schema <- infer_nanoarrow_schema(array)
+  geo <- geoarrow_schema_parse(schema)
+  vctr <- as_geoarrow_vctr(array)
+
+  if (geo$extension_name == "geoarrow.wkt") {
+    out <- wk::new_wk_wkt(convert_array(force_array_storage(array)))
+  } else {
+    out <- wk::wk_handle(vctr, wk::wkt_writer())
+  }
+
+  wk::wk_crs(out) <- wk::wk_crs_output(vctr, to)
+  wk::wk_is_geodesic_output(vctr, to)
+  out
```

Comment:
Was the intent to copy geodesic from `to`?

```suggestion
  wk::wk_is_geodesic(out) <- wk::wk_is_geodesic_output(vctr, to)
```

Will need to add a couple of tests for checking geodesic TRUE & FALSE